### DISCUSSION TITLE
I can't get the multi resin exposure to work with the Uniformation GKTwo

### DISCUSSION THREAD

**Discussion by DrSteve-Brule** [sn4k3/UVtools#740] [Q&A] [answered]
Hello Everyone,

First of all I 'd like to express my gratitude for this software and it intended uses in the 3d printing community.

Here is my problem

I'd like to make a multi resin exposure print for my Uniformation GKTwo. For the life of me I can't figure out how to get it to work. THe Best I could get it to do, it prints each layer on each test individually. I'd like it to expose each layer on all the 'parts' simultaneously and turn off the screen sequentially for each 'part' that was specified to have a shorter exposure time. That's the way the anycubic RERF test works anyway.

Currently, when the build plate lifts up and then tries to start a layer on a different part, it fails. Also, I've noticed that I can't export the part as a jxs file, and only as a Chitu file. Does that have something to do with this?

Anyone have experience with the GKTwo?

**Comment by sn4k3** [comment]
Uniformation GKTwo JXS is the Encrypted CTB format under a renamed file extension (ie, you can rename ctb to jxs and it will also print), so they are the same format under the mess of JXS is a zip on GKone, they should not have used same extension for different formats...

Many users are getting trouble with that test when on that new CTB format, can't discover why since I have no machine to debug. UVtools is the only software who take advantage of such "advanced" approach and so it's even harder to get the details why it fails. Maybe a flag or simply they don't allow to return to a lower height, I don't know.

**Comment by DrSteve-Brule** [comment]
I noticed that I cannot print a part flat on the build plate as well. It only works with a pad and supports. Strange.

**Reply by sn4k3** [reply]
Try to print flat without process with UVtools

**Reply by DrSteve-Brule** [reply]
So I tried printing flat and couldn't get it to stick to the build plate. I just got in some ACF film and it sticks to the build plate now.

**Reply by sn4k3** [reply]
Looks like your problem is with plate adhesion. 
If your plat is smooth as silk that is the problem. Sand the plate with 80-grit and relevel the printer will give you a sticky plate. That with 20s bottom exposure and prints will stick like a rock and you will have a hard time removing them.

**Comment by DrSteve-Brule** [comment]
Thank you for your feedback! 

Was wondering if the multi exposure work now on the Uniformation GKTwo. 

Best

**Comment by CalebBarnes** [comment]
Weird, I can't even get it to start the multiple exposure finder test print on my GKTwo. The screen just shows file error then wont start.

**Reply by sn4k3** [reply, marked as answer]
Try to export the file to encrypted CTB. Gktwo format is a mess.

**Reply by LucaBastelt** [reply]
Commenting for all google arrivals: This worked for me to get the UVTools exposure finder working on my GKTwo. Export as CTB for GKTwo instead of saving as CTB. This creates a .gktwo.ctb
